test/unittest/tests/test_tailq.cpp

```diff
@@ -0,0 +1,134 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <list>
+#include <catch2/catch.hpp>
+#include "wt_internal.h"
+
+template <class T> struct tailq_entry {
+    explicit tailq_entry<T>(T const &value) : _value(value){};
+    TAILQ_ENTRY(tailq_entry<T>) _queue;
+    T _value;
+};
+
+template <class T> class TestTailQWrapper {
+    public:
+    TestTailQWrapper();
```

Comment:
Same question, do we need them? Should we make them `= default`?

Reply:
No, these ones do some actual work.

Reply:
My bad, thanks!